modules/tinymce/src/core/main/ts/annotate/AnnotationChanges.ts

```diff
@@ -6,10 +6,12 @@
  */
 
 import { Arr, Cell, Obj, Optional, Optionals, Singleton, Throttler } from '@ephox/katamari';
+import { Attribute } from '@ephox/sugar';
 
 import Editor from '../api/Editor';
 import { AnnotationsRegistry } from './AnnotationsRegistry';
-import { identify } from './Identification';
+import { findMarkers, identify } from './Identification';
+import { dataAnnotationActive } from './Markings';
```

Comment:
```suggestion
import * as Markings from './Markings';
```
We generally prefer the `* as Name` format as it helps make things more readable. Up to you if we do this here. We also probably should do it for `Identification` if we do I guess, but I can't leave a suggestion for that sorry.

Reply:
It also makes it way more verbose. But I guess it makes it easier to see if the thing is local or not.

Reply:
I guess it make sense some times but not always since if there was a bunch of types that needed to be imported you don't want to have Markings.MyType etc all over. I guess for alloy we have that but that is because it's so many types and that is the only way you can export things in submodules in js though a single entry point.